src/app/drivingLicence/fields.js

```diff
@@ -53,7 +53,15 @@ module.exports = {
     validate: [
       "required",
       "date",
-      { type: "before", arguments: [new Date().toISOString().split("T")[0]] },
+      { type: "before", arguments: [
```

Comment:
Can you just double check these fields (except for issue/expirydate) are behaving as expected now with this change?

Reply:
Fixed.